packages/venia-ui/lib/components/ProductFullDetail/productFullDetail.js

```diff
@@ -45,47 +47,55 @@ const ProductFullDetail = props => {
     ) : null;
 
     return (
-        <Form className={classes.root}>
-            <section className={classes.title}>
-                <h1 className={classes.productName}>{productDetails.name}</h1>
-                <p className={classes.productPrice}>
-                    <Price
-                        currencyCode={productDetails.price.currency}
-                        value={productDetails.price.value}
+        <Fragment>
+            <Breadcrumbs
+                categoryId={breadcrumbCategoryId}
+                currentProduct={productDetails.name}
+            />
```

Comment:
This and the returned prop are the only changes. Everything just shifted cause of the `<Fragment>`.